cypress/integration/Regression/CreateModels/AllEntityTypes.spec.js

```diff
@@ -41,9 +58,15 @@ describe('All Entity Types - CreateModels', () => {
       entities.CreateNewEntity({ type: 'Programmatic', name: 'programmaticMultiValuedEntity', multiValued: true})
     })
 
+    // Alternate testing of multiValued and negatable on different days 
+    // so that we test these in combination regularly.
```

Comment:
Why does it need to be random or based on day of year?

If it's only there to get more coverage just add more tests. The tests should be consistent and not matter which day you choose to run them. I'm imagining explaining to someone that they can only test that case on even days of the year and them being in disbelief.  This just adds complexity to think about.

I think last time I looked these tests take ~7 seconds to create each entity and total time was ~2 minutes, 25 seconds on a tests that are highly unlikely to fail and delay info about the more important tests. Another reason to split into smoke / remaining.

I suppose there are shortcuts like importing a model with all variations of entities which would implicitly test server and luis, and the UI test could only verify a few of the type and dropdown functionality but I'm hesitant to do that either.

Reply:
I agree with most of what you are saying here. I was trying to increase coverage while not impacting the test run time as much. But we have better ways of dealing with this so...

Since we are at the point of needing to break tests into Smoke vs Full/Regression Testing I'll be sure this test is in the full test list and fix it to test every possible combination when that change over is made.